c/src/sbp.c

```diff
@@ -323,6 +384,12 @@ s8 sbp_process(sbp_state_t *s, s32 (*read)(u8 *buff, u32 n, void *context))
     if (1 == rd)
       if (temp == SBP_PREAMBLE) {
         s->n_read = 0;
+        /* set frame_buff to 0 after each preamble.
+         * Note, library functions are not used to avoid more dependencies. */
+        for (int i = 0; i < 255; i++) {
```

Comment:
`255` doesn't seem right here. since it's a static buffer we can use `sizeof(s->frame_buff)`